src/main/resources/db/changelog/changes/v1.0.3/sql/create-list-versions-table.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS list_versions
+(
+    id                      UUID            NOT NULL PRIMARY KEY,
+    name                    VARCHAR(255)    NOT NULL,
+    description             VARCHAR(1024),
+    entity_type_id          UUID            NOT NULL,
+    fql_query               VARCHAR(1024),
+    user_friendly_query     VARCHAR(1024),
```

Comment:
I'd change these to just `TEXT`, unless they are `VARCHAR(1024)`s in the list_details table (and if that's the case, we should definitely create a ticket to fix that).